Makefile

```diff
@@ -74,8 +74,13 @@ $(OUT)/trivialDAO_storage.tz : now_val = `date +"%s"`
 $(OUT)/trivialDAO_storage.tz : metadata_map = (Big_map.empty : metadata_map)
 $(OUT)/trivialDAO_storage.tz : fixed_proposal_fee_in_token = 0n
 $(OUT)/trivialDAO_storage.tz : ledger = ([] : ledger_list)
-$(OUT)/trivialDAO_storage.tz : quorum_threshold = {numerator = 1n; denominator = 10n}
+$(OUT)/trivialDAO_storage.tz : quorum_threshold = { numerator = quorum_denominator/10n }
```

Comment:
This way it's going to be quite hard to provide a custom value here without knowing about implementation details like `quorum_threshold`, perhaps we should give an easier way to give these fractions?

Maybe only accept a `nat` for a percentage here and do the math in the LIGO code?

Reply:
Have updated to accept percentages in `nat` for these values.